opencensus/stats/aggregation_data.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, sum_data):
         super(SumAggregationDataFloat, self).__init__(sum_data)
         self._sum_data = sum_data
 
-    def add_sample(self, value):
+    def add_sample(self, value, timestamp, attachments):
```

Comment:
Is the `timestamp` and `attachments` used anywhere here?

Reply:
They are not used, but are needed to maintain the def signature. I changed these parameter to be optional.